src/lib/Libwin/passwd.c

```diff
@@ -4817,15 +4817,15 @@ logon_pw(char *username,
 	char msg[4096])
```

Comment:
Please declare it as a character pointer for the sake of consistency with the rest of the PBS Pro code.